Textile/Classes/AccountApi.m

```diff
@@ -28,6 +28,9 @@ - (NSData *)decrypt:(NSData *)data error:(NSError * _Nullable __autoreleasing *)
 
 - (Contact *)contact:(NSError * _Nullable __autoreleasing *)error {
   NSData *data = [self.node accountContact:error];
+  if (*error) {
+    return nil;
+  }
```

Comment:
I have to explicitly check there error and not continue if there is one. Otherwise, the init of the `Contact` below will happily create a default `Contact` from `nil` `NSData` and reset the error pointer to `nil` and the caller has no idea there was an error, but their `Contact` is missing any useful data.